src/__tests__/lib/rules/prefer-in-document.js

```diff
@@ -0,0 +1,51 @@
+/* eslint-disable no-template-curly-in-string */
```

Comment:
curious why we had to disable this?

Reply:
I think that was just left from duplicating one of the other rules. I've removed it now.